README.md

```diff
@@ -35,7 +35,7 @@ This appender uses [BigQueue](https://github.com/bulldog2011/bigqueue) implement
 ### Parameters
 | Parameter          | Default                              | Explained  |
 | ------------------ | ------------------------------------ | ----- |
-| **token**              | *None*                                 | Your Logz.io token, which can be found under "settings" in your account |
+| **token**              | *None*                                 | Your Logz.io token, which can be found under "settings" in your account, If the value begins with `$` then the the appender look for environment variable with the name specified. For example: `$LOGZIO_TOKEN`will look for env variable named `LOGZIO_TOKEN` |
```

Comment:
If the value begins with `$` then the appender looks for an environment variable with the name specified. For example: `$LOGZIO_TOKEN` will look for environment variable named `LOGZIO_TOKEN`

And can you please also add a documentation for logzioUrl ?